CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-3.0.0
+4.0.0
+-----
+
+- **Major**: Improve Payplug setSecretKey with init method
+- **NEW**: Remove Payplug::setSecretKey function
+- **NEW**: Excluded configuration API_VERSION versionning, it will be send from the apps nowadays
```

Comment:
Ça aussi c'est **MAJOR**, et pour plus de clarté je pense qu'il serait préférable de mettre quelque chose du genre “API version must now be specified explicitly. ”

Reply:
Noté.